_data/datasets/education-budget-brief-2016.yaml

```diff
@@ -1,84 +0,0 @@
-author:
```

Comment:
This doesn't exist in ckan. I suspect it was deleted.

The rebuild for contributed data doesn't currently clean these up, we just stop linking to it.